actions/host_get.py

```diff
@@ -33,14 +35,18 @@ def run(self, host_ids, host_names, vsphere=None):
         Returns:
         - dict: Host network hints details.
         """
+        self.establish_connection(vsphere)
 
-        # TODO review using propertspec for retrieving all Hosts's at onces.
-        results = {}
+        results = None
         if not host_ids and not host_names:
-            raise ValueError("No IDs nor Names provided.")
```

Comment:
i would rather have a separate parameter here to request all hosts, rather than changing the behavior / error condition that others may rely on when no host is passed in

Reply:
Should be good to go now